NewBank/newbank/server/Customer.java

```diff
@@ -8,6 +8,7 @@
 
     private String password;
     private int UID;
+    private int numberOfAccounts = 0;
```

Comment:
Is this needed? Can't we just use accounts.size()? 
This is data duplication, and it's an easy way to introduce inconsistencies in the system. Imagine that someone deletes an account in the future and forgets to decrease the counter.